doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -289,6 +289,15 @@ Other libraries
     The library is no longer directly referenced from the Event Manager.
     Instead, it uses the Event Manager hooks to connect with the manager.
 
+Libraries for Zigbee
+--------------------
+
+* :ref:`lib_zigbee_shell`:
+
```

Comment:
What about an entry for the whole group of ``zcl groups`` shell commands?

Reply:
I don't quite understand what you mean, could you elaborate?

Reply:
done